src/Microsoft.Health.Fhir.Liquid.Converter/Json/JsonDataParser.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Liquid.Converter.Exceptions;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.Json
+{
+    public class JsonDataParser
+    {
+        public IDictionary<string, object> Parse(string json)
+        {
+            if (string.IsNullOrWhiteSpace(json))
```

Comment:
>IsNullOrWhiteSpace [](start = 23, length = 18)

shall we allow empty?

Reply:
Currently null, empty or string consists only of white-space characters are not allowed. Do we have any scenario that may accept empty input?